README.md

```diff
@@ -371,6 +371,20 @@ if let foo = foo {
 }
 ```
 
+Or when unwrapping multiple optionals, prefer this:
+
+```swift
+if 
+    let foo = foo,
```

Comment:
Just a personal preference that the `let`s be lined up vertically. You'd prefer something like this?
```swift
if let foo = foo,
    let bar = bar {
        foo.go()
}
```